### PR TITLE
Use environment appName instead of hardcoded Loop

### PR DESCRIPTION
**Pull request by marionbarker**
1. Add code needed to use environment that ties appName to CFBundleDisplayName
2. Use the appName from the environment to replace hardcoded Loop in strings
   * For one file, I did not know how to insert the environment variable for appName, so I put in a work-around string instead
   * This can be updated if someone provides instructions (`PumpOpsSession.swift`)

None of these changes could be tested.
* I think some of these views just aren't used anymore
* The one that is used requires an MDT with MySentry to test and I only have an MDT

### CODE REVIEW

File: MinimedKitUI/RadioSelectionTableViewController.swift [hunk 1]
```diff
@@ -26,21 +27,23 @@ extension RadioSelectionTableViewController: IdentifiableClass {
...
+        vc.contextHelp = String(format: LocalizedString("Alkaline and Lithium batteries decay at differing rates. Alkaline tend to have a linear voltage drop over time whereas lithium cell batteries tend to maintain voltage until halfway through their lifespan. Under normal usage in a Non-MySentry compatible Minimed (x22/x15) insulin pump running %1$@, Alkaline batteries last approximately 4 to 5 days. Lithium batteries last between 1-2 weeks. This selection will use different battery voltage decay rates for each of the battery chemistry types and alert the user when a battery is approximately 8 to 10 hours from failure.", comment: "Instructions on selecting battery chemistry type (1: appName)"), appName)
 
         return vc
     }
 
     static func useMySentry(_ value: Bool) -> T {
         let vc = T()
+        @Environment(\.appName) var appName
```
Comment (ps2) [inline MinimedKitUI/RadioSelectionTableViewController.swift:41]: @Environment is generally used on a SwiftUI view (this is an extension on a UIKit View controller), as the data is passed through the view hierarchy.  I'm not sure how/if this is working?

**Comment by marionbarker** [comment]
I prepared code to use appName based off a few hints. I am working via "prototype example" and don't have a solid understanding of how and why it works. Your comment above that "@Environment is generally used on a SwiftUI view (this is an extension on a UIKit View controller)" probably indicates this will not work for `MinimedKitUI/RadioSelectionTableViewController.swift` as desired.

I was not able to test the modification for this file because I do not have a Medtronic pump that supports MySentry. I was able to test the other instances and show that they did work as expected.

There was a case for OmniBLE/Kit (BeepPreference) where the attempt to use appName did not succeed. See this link:
* https://github.com/LoopKit/OmniBLE/pull/120#issuecomment-2053694043

To summarize tests done in that PR: "To reiterate the problem, this picks up the defaultValue from LoopKit instead of the appName from the environment that I expected."

In that case, I demonstrated that whatever value was found on this line in `LoopKit/LoopKitUI/Extensions/Environment+AppName.swift` is what is inserted into appName when I tried to use it.
* https://github.com/LoopKit/LoopKit/blob/dev/LoopKitUI/Extensions/Environment%2BAppName.swift#L14
